Modules/Core/Helpers/trans_helper.php

```
@@ -22,40 +19,42 @@
-        $current_language = $CI->session->userdata('user_language');
-
-        if (empty($current_language) || $current_language == 'system') {
-            // todo gives error at startup, fix later
-            // #1034: Translation breaks in PDF-template
-            $current_language = get_setting('default_language') ?? 'english';
-        }
-
-        // Load the default language and translate the string
-        set_language('english');
+if (!function_exists('trans')) {
+    function trans($line, ?string $id = '', $default = null)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)